examples/crack_branching.cpp

```diff
@@ -62,26 +73,35 @@ int main( int argc, char* argv[] )
         Kokkos::Array<Kokkos::Array<double, 3>, 1> notch_positions = { p01 };
         CabanaPD::Prenotch<1> prenotch( v1, v2, notch_positions );
 
-        // Choose force model type.
+        // ====================================================
+        //                    Force model
+        // ====================================================
         using model_type =
             CabanaPD::ForceModel<CabanaPD::PMB, CabanaPD::Fracture>;
         model_type force_model( delta, K, G0 );
 
-        // Create particles from mesh.
+        // ====================================================
+        //                  Grid generation
```

Comment:
I find this confusing since the background grid is never actually built

Reply:
Done